### ISSUE TITLE
Is there support for elasticsearch 8.x.x?

### ISSUE THREAD

**Issue by israellias** [django-es/django-elasticsearch-dsl#411]
The documentation mentions 7.x.x and earlier. 
But I'm having issues with elasticsearch 8. I want to make clear the compatibility

**Comment by violuke** [comment]
Yeah, we tried upgrading ES to 8.2 and get:

```
ConnectionError

('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
```

It looks like elasticsearch-dsl also needs upgrading and I suspect the upgrade of that package will solve the issues here.... hopefully without this package needing any significant changes

https://github.com/elastic/elasticsearch-dsl-py/issues/1569

**Comment by sethmlarson** [comment]
@violuke The error you've described is due to connecting to a cluster with HTTPS enabled via HTTP, not anything to do with 8.x versus 7.x. In 8.x and later HTTPS is the default setting for Elasticsearch.

**Comment by violuke** [comment]
elasticsearch-dsl v8.9.0 with official support for ESv8 is finally out :tada: https://github.com/elastic/elasticsearch-dsl-py/releases/tag/v8.9.0

Could we get an update to this package to allow installing alongside django-elasticsearch-dsl?

**Comment by oehrlein** [comment]
Just submitted a PR for this (#458).

**Comment by pquentin** [comment]
Thanks @oehrlein! I'm glad that my work on elasticsearch-dsl allowed you to support 8.x here too.

Can this be closed now that #458 was merged and 8.0 released? https://github.com/django-es/django-elasticsearch-dsl/releases/tag/8.0

**Comment by oehrlein** [comment]
@pquentin No problem!

I don't have sufficient privileges to do so, so @israellias, @safwanrahman, or @saadmk11 will have to.
